content/docs/guides/remix.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Connect a Remix application to Neon
+subtitle: Set up a Neon project in seconds and connect from a Remix application
+enableTableOfContents: true
+updatedOn: '2023-10-06T17:44:14.691Z'
+---
+
+Remix is a full stack web framework that lets you focus on the user interface and work back through web standards to deliver a fast, slick, and resilient user experience. People are gonna love using your stuff.
+
+
+To create a Neon project and access it from a Remix application:
+
+1. [Create a Neon project](#create-a-neon-project)
+2. [Create a Remix project and add dependencies](#create-a-remix-project-and-add-dependencies)
+3. [Configure a Postgres client](#configure-the-postgres-client)
+4. [Run the app](#run-the-app)
+
+## Create a Neon project
+
+If you do not have one already, create a Neon project. Save your connection details including your password. They are required when defining connection settings.
+
+1. Navigate to the [Projects](https://console.neon.tech/app/projects) page in the Neon Console.
+2. Click **New Project**.
+3. Specify your project settings and click **Create Project**.
+
+## Create a Remix project and add dependencies
+
+1. Create a Remix project if you do not have one. For instructions, see [Quick Start](https://remix.run/docs/en/main/start/quickstart), in the Remix documentation.
+
+2. Add project dependencies using one of the following commands:
+
+    <CodeTabs labels={["node-postgres", "postgres.js", "Neon serverless driver"]}>
+
+      ```shell
+      npm install pg
+      ```
+
+      ```shell
+      npm install postgres
+      ```
+
+      ```shell
+      npm install @neondatabase/serverless
+      ```
+
+    </CodeTabs>
+
+## Store your Neon credentials
+
+Add a `.env` file to your project directory and add your Neon connection string to it. You can find the connection string for your database in the **Connection Details** widget on the Neon **Dashboard**. For more information, see [Connect from any application](/docs/connect/connect-from-any-app).
+
+<CodeBlock shouldWrap>
+
+```shell
+DATABASE_URL=postgres://[user]:[password]@[neon_hostname]/[dbname]
+```
+
+</CodeBlock>
+
+## Configure the Postgres client
+
+TODO. Add explanation for why a `.db.server` file is required.
+
+### db.server
+
+Create a `db.server.ts` file at the root of your `/app` directory add the following code snippet to connect to your Neon database:
```

Comment:
Totally just a suggestion since you have this TODO.

```suggestion
### db.server

Create a `db.server.ts` file at the root of your `/app` directory. The [`*.server.ts` file convention](https://remix.run/docs/en/main/file-conventions/-server) ensures that Remix won't add this code into the client bundle. Add the following code snippet to connect to your Neon database:
```

Reply:
That's great! Thanks @brookslybrand